packages/components/src/range-control/index.js

```diff
@@ -29,7 +29,15 @@ function RangeControl( {
 	...props
 } ) {
 	const id = `inspector-range-control-${ instanceId }`;
-	const onChangeValue = ( event ) => onChange( Number( event.target.value ) );
+	const resetValue = () => onChange();
+	const onChangeValue = ( event ) => {
+		const newValue = event.target.value;
+		if ( newValue === '' ) {
```

Comment:
Shouldn't you apply Number to cast it to int first and then check if it is greater than 0?

Reply:
I bet in IE11 you can type `abc` 😢

Reply:
What if the value can be below `0`? Assuming ranges are always above zero causes bugs like #9254.

Reply:
We may have a RangeControl where 0 is an acceptable input, that's why I only apply the reset when an empty input is passed.

Reply:
I just tested in IE and you can type `abc` but it gets reset to empty on this branch, seems fine.

Reply:
It's not only EI11 even chrome only enforces min and max when input numbers using the arrows but allows non conformant numbers to be typed by the user. Maybe browsers have some reason for this and it looks like this component should just be a wrappper around the browser implementation. In this case the bug was our fault because casting an empty string to a number sets a 0 value and we did not checked for that.